src/commands/general/guildInfo.ts

```diff
@@ -0,0 +1,19 @@
+import Command, { SendFunction } from '../../util/BaseCommand';
+import SentinelClient from '../../client/SentinelClient';
+import { Message } from 'discord.js';
+import CommandArguments from '../../util/CommandArguments';
+
+export default class guildStatsCommand extends Command {
+	constructor(client: SentinelClient) {
+		super(client, {
+			aliases: ['gs', 'serverinfo', 'guildinfo', 'serverstats'],
+			name: 'guildstats',
+			dmAllowed: false,
+			description: 'Display your guild information.'
```

Comment:
```suggestion
			description: 'Displays information about the current server.'
```
better for the average user to understand, as well as more accurate